lib/wash/exe/chrome.js

```diff
@@ -13,16 +13,112 @@
 // limitations under the License.
 
 import AxiomError from 'axiom/core/error';
+import Completer from 'axiom/core/completer';
 import chromeAgentClient from 'wash/chrome_agent_client';
 
 /** @typedef JsExecuteContext$$module$axiom$fs$js$execute_context */
 var JsExecuteContext;
 
+/** @const @private */
+var STDIN_ = '$STDIN';
+/** @const @private */
+var STDIN_RE_ = new RegExp(STDIN_);
+/** @const @private */
+var DEFAULT_TABS_FOR_LIST_ = 'all';
+/** @const @private */
+var DEFAULT_TIMEOUT_ = 1000;
+/** @const @private */
+var DEFAULT_RUN_AT_ = 'idle';
+
+var HELP_ = [
+  'SYNOPSYS:',
+  '',
+  'chrome -a|--call-api <API method> [<API arg>...] [OPTIONS]',
+  '  Executes a Chrome Extensions API call on the browser side and returns the',
+  '  result. The API arguments must be whitespace-separated.',
+  '',
+  'chrome -s|--exec-script <script> TABS [OPTIONS] [MORE_OPTIONS]',
+  '  Executes a script in the top frame or all frames of the specified tab(s),',
+  '  returning the results as a map {tabId: result/error, ...}.',
+  '',
+  'chrome -c|--insert-css <css> TABS [OPTIONS] [MORE_OPTIONS]',
+  '  Injects a CSS fragment into the top frame or all frames of the specified',
+  '  tab(s). Returns an empty map if all insertions were successful, or',
+  '  a map {tabID: error, ...} in case there were errors.',
+  '',
+  'chrome -l|--list-tabs [TABS] [OPTIONS] [MORE_OPTIONS]',
+  '  Prints a compact list of requested tabs\' IDs and titles for',
+  '  a quick identification. For a more involved querying and a detailed',
+  '  list of tab properties, use:',
+  '  `chrome --exec-script tabs.query ...`.',
+  '',
+  'chrome -d|--api-doc [<API or API method>...]',
+  '  Opens the Chrome Extensions API documentation for the given API',
+  '  or API method in a new tab.',
+  '',
+  'chrome --install-agent',
+  '  Initiates installation of the Chrome Agent extension by opening its',
+  '  Chrome Web Store page in a new tab.',
+  '',
+  'OPTIONS:',
+  '',
+  '--timeout <millisec>',
+  '  A timeout in milliseconds, defaults to ' + DEFAULT_TIMEOUT_ + 'ms.',
+  '  Prevents hanging in presence of rogue tabs, e.g. tabs in the "Aw, snap!"',
+  '  state.',
+  '',
+  '-n|-no-trailing-eol',
+  '  Do not print the trailing newline character. Useful when the output',
+  '  is piped into another command.',
+  '',
+  'TABS:',
+  '',
+  '-t|--tabs <tab ID>,...|all|window',
+  '  Specifies a set of tabs to apply the action to.',
+  '  "all" means "all windows"; "window" means "current window"; if IDs are',
+  '  used, they must be comma-separated. Note that for --list-tabs, this',
+  '  argument is optional and defaults to "' + DEFAULT_TABS_FOR_LIST_ + '".',
+  '',
+  'MORE_OPTIONS:',
```

Comment:
I think `PAGE_OPTIONS` sounds cryptic. OTOH, I couldn't find any term that would aptly group these together, so I've renamed them to `SCL_OPTIONS` (because they apply to -s, -c, -l modes only) :)